static/css/common.css

```diff
@@ -753,4 +753,27 @@ a:active {
 }
 .static label .int-he .sub{
   margin-right: 4px;
+}
+.categoryColorLineRainbow {
+    top: 60px;
+    left: 0;
+    width: 100%;
+    height: 4px;
+    z-index: 101;
+    background: linear-gradient(
+    90deg,
+    var(--tanakh-teal) 0% 10%,
```

Comment:
I'm realizing now, do these variables exist in common.css? Does this work on the static page that doesn't have s2.css?

Reply:
it works, seems like they are global
![image](https://github.com/user-attachments/assets/91c32d91-72ba-47d6-a7eb-2b5cfe12cc53)